### PR TITLE
Update dutch language

### PR DESCRIPTION
**Pull request by TomG777**
Update (and reformat) the dutch language file to the likes of the latest English version.

### CODE REVIEW

**Review by inxomnyaa — COMMENTED** [review]
Waiting for @sandertv to check back

File: resources/lang/nld.ini [hunk 1]
```diff
@@ -1,46 +1,111 @@
...
-runingame = "Voer deze operatie alstublieft in-game uit"
 commands = "Operaties"
-; user interfaces
-ui.confirmation = "Confirmatie"
-ui.confirmation.yes = "Ja"
-ui.confirmation.no = "Nee"
-; ui brush
+enabled = "ingeschakeld"
+disabled = "uitgeschakeld"
+confirmation = "Confirmatie"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: Better translated as 'Bevestiging'

File: resources/lang/nld.ini [hunk 2]
```diff
@@ -1,46 +1,111 @@
...
+confirmation.yes = "Ja"
+confirmation.no = "Nee"
+; errors
+error = "Er is iets fout gegaan"
+error.command-error = "Het lijkt erop dat je een argument mist of de opdracht verkeerd hebt gebruikt!"
+error.runingame = "Voer deze operatie alstublieft in-game uit"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: 'operatie' might be better translated as 'opdracht'.

File: resources/lang/nld.ini [hunk 3]
```diff
@@ -1,46 +1,111 @@
...
+error.limitexceeded = "U probeert te veel blokken tegelijk te bewerken. Verklein de selectie of verhoog de limiet"
+error.notarget = "Geen doelblok gevonden. Vergroot gereedschapsbereik met //setrange indien nodig"
+error.noselection = "Geen selectie gevonden - selecteer eerst een gebied"
+error.selectioninvalid = "De selectie is niet geldig! Controleer of alle pisities zijn geselecteerd!"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: 'pisities' -> 'posities'

File: resources/lang/nld.ini [hunk 4]
```diff
@@ -1,46 +1,111 @@
...
+error.nosession = "Er is geen sessie gemaakt - waarschijnlijk geen toestemming om {% 0} te gebruiken"
+error.noclipboard = "Geen klembord gevonden - maak eerst een klembord"
+warning.differentlevel = "[WAARSCHUWING] Je bewerkt een niveau waar je momenteel niet in zit!"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: 'level' is meant as 'world' in this context. 'niveau' should be 'wereld'.

File: resources/lang/nld.ini [hunk 5]
```diff
@@ -1,46 +1,111 @@
...
+; commands
+command.info.title = "Informatie"
+command.limit.current = "Huidig limiet: {%0}"
+command.limit.set = "Limiet voor wijziging van blok is ingesteld op {%0}"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: Might be better translated as 'Limiet voor aantal wijzigingen van blokken is ingesteld op {%0}'.

File: resources/lang/nld.ini [hunk 6]
```diff
@@ -1,46 +1,111 @@
...
+command.setrange.current = "Huidig bereik: {%0}"
+command.setrange.set = "Gereedschapsbereik is ingesteld op {%0}"
+command.biomeinfo.attarget = "Biome op doel"
+command.biomeinfo.atposition = "Biome op positie"
+command.biomeinfo.result = "{%0} biomen gevonden in selectie"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: 'biomen' might be better kept as a full English word ('biomes').

File: resources/lang/nld.ini [hunk 7]
```diff
@@ -1,46 +1,111 @@
...
+command.biomelist.result.line = "ID: {%0} Naam: {%1}"
+command.brushname.set = "Kwast naam ingesteld op \"{%0}\""
+command.clearclipboard.cleared = "Klemborden gewist"
+command.flip.try = "Klembord proberen te draaien met {%0}"
+command.flip.success = "Klembord succesvol gedraaid"
+command.rotate.try = "Probeeren het klembord met {%0} graden te draaien"
+command.rotate.success = "Klembord succesvol gedraaid"
+command.history.cleared = "Geschiedenis gewist"
+command.listchunks.found = "{%0} chunks in selectie gevonden"
+command.size = "Selectie formaat"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: 'formaat' might be better translated as 'grootte'.

File: resources/lang/nld.ini [hunk 8]
```diff
@@ -1,46 +1,111 @@
...
+; session
+session.undo.none = "Niks om ongedaan te maken"
+session.undo.left = "U kunt nog {%0} acties ongedaan maken"
+session.redo.none = "Niets om opnieuw te doen"
+session.redo.left = "U heeft nog {%0} opnieuw uit te voeren acties"
+session.brush.added = "{%0} toegevoegd aan sessie"
+session.brush.deleted = "Verwijderd {%0} (UUID {%1})"
+session.brush.removed = "Verwijderd {%0} (UUID {%1})"
+session.language.set = "Taal succesvol ingesteld op {%0}"
+session.language.notfound = "Taal {% 0} niet gevonden, reset naar standaard"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: 'reset' -> 'gereset'

File: resources/lang/nld.ini [hunk 9]
```diff
@@ -1,46 +1,111 @@
...
+; task
+task.copy.success = "Async Copy geslaagd, kostte {% 0}, kopieerde {% 1} blokken van {% 2}."
+task.count.success = "Async-analyse slaagde, kostte {% 0}"
+task.count.result = "{% 0} blokken gevonden van in totaal {% 1} blokken"
+task.fill.success = "Async Fill is geslaagd, kostte {%0}, {%1} blokken uit {%2} gewijzigd."
+task.replace.success = "Async vervangen geslaagd, kostte {%0}, {%1} blokken van {%2} gewijzigd."
+task.revert.undo.success = "Async Ongedaan maken geslaagd, kostte {%0}, {%1} blokken uit {%2} gewijzigd."
+task.revert.redo.success = "Async opnieuw doen is geslaagd, kostte {%0}, {%1} blokken uit {%2} gewijzigd."
```
Comment (Sandertv) [inline resources/lang/nld.ini]: 'kostte' in these sentences might be better translated as 'duurde'.

File: resources/lang/nld.ini [hunk 10]
```diff
@@ -1,46 +1,111 @@
...
+flags.hollowwclosed = "Hol met gesloten einden"
+flags.natural = "Natuurlijk"
+; tools
+; wand tool
+tool.wand = "Toverstaf"
+tool.wand.lore.1 = "Klik met de linkermuisknop op een blok om de positie 1 van een selectie in te stellen"
+tool.wand.lore.2 = "Klik met de rechtermuisknop op een blok om de positie 2 van een selectie in te stellen"
+tool.wand.lore.3 = "Gebruik //togglewand om van functionaliteit te wisselen"
+tool.wand.disabled = "De toverstaf is uitgeschakeld. Gebruik //togglewand om het opnieuw in te schakelen"
+tool.wand.setenabled = "Het gereedschap Toverstaf is nu{%0}!"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: Missing space between 'nu' and '{%0}'

File: resources/lang/nld.ini [hunk 11]
```diff
@@ -1,46 +1,111 @@
...
+tool.debug.lore.1 = "Klik met de linkermuisknop op een blok om informatie te krijgen"
+tool.debug.lore.2 = "zoals de naam en schadewaarden van een blok"
+tool.debug.lore.3 = "Gebruik //toggledebug om van functionaliteit te wisselen"
+tool.debug.disabled = "Het foutopsporing gereedschap is uitgeschakeld. Gebruik //toggledebug om het opnieuw in te schakelen"
+tool.debug.setenabled = "Het foutopsporing gereedschap is nu {%0}!"
+; flood tool
+ui.flood.title = "Opvullings menu"
+ui.flood.options.limit = "Maximum aantal blokken"
+ui.flood.options.blocks = "Blokken"
+ui.flood.options.blocks.placeholder = "Blokken onderbroken door semicolons"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: semicolon = puntkomma

File: resources/lang/nld.ini [hunk 12]
```diff
@@ -1,46 +1,111 @@
...
-ui.flood.options.limit = "Maximum aantal blokken"
-ui.flood.options.blocks = "Blokken"
-ui.flood.options.blocks.placeholder = "Blokken onderbroken door semicolons"
-ui.flood.options.label.infoapply = "Klik op de "Submit" knop om het op te sturen"
\ No newline at end of file
+ui.brush.options.flags = "Opties toevoegen?"
+; language
+ui.language.title = "Select language"
+ui.language.label = "Set the language of your session. If your language is not available, you may translate the plugin on GitHub!"
+ui.language.dropdown = "Select a language"
```
Comment (Sandertv) [inline resources/lang/nld.ini]: These strings aren't translated. Are they meant to be?

**Review by Sandertv — CHANGES_REQUESTED** [review]

File: resources/lang/nld.ini (hunk 8 above)
Comment (inxomnyaa) [reply resources/lang/nld.ini]: and please remove the space in {%0}

File: resources/lang/nld.ini (hunk 6 above)
Comment (inxomnyaa) [inline resources/lang/nld.ini:31]: Could be similar to another term we discussed in german translation (Meta-Data), it is kind of specific. I guess @snadertv has a good point

File: resources/lang/nld.ini (hunk 12 above)
Comment (inxomnyaa) [reply resources/lang/nld.ini]: Indeed, they must be translated (maybe a logic mistake by me - maybe it should default to the server default language, but we need the translations anyways)

File: resources/lang/nld.ini (hunk 6 above)
Comment (Sandertv) [reply resources/lang/nld.ini:31]: Nevermind; This is fine.

File: resources/lang/nld.ini (hunk 12 above)
Comment (thebigsmileXD) [reply resources/lang/nld.ini]: Indeed, they must be translated (maybe a logic mistake by me - maybe it should default to the server default language, but we need the translations anyways)

**Comment by TomG777** [comment]
Sorry for the delay but I have implemented the suggestions/corrections by Sander. I guess I should have waited with the PR and have proof-read it again after a break.

I based the 'operatie' on the already present commands but I agree that 'opdracht' flows better.
the rest were just being unfamiliar with the tool or my sloppiness.

**Comment by inxomnyaa** [comment]
> I guess I should have waited with the PR and have proof-read it again after a break.

That is what PR's are for ;)

Thanks for the changes

**Comment by inxomnyaa** [comment]
@Sandertv Is this looking fine to you?

**Review by Sandertv — APPROVED** [review]

**Pull request merged**